css/style.css

```diff
@@ -0,0 +1,124 @@
+html,
+body {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+}
+
+@media screen and ( max-width: 600px ) {
+    body {
+        font-size: 10px;
+    }
+}
+
+
+#navbar {
+    border: 4px solid brown;
+    display: flex;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    height: 60px;
+    background-color: rgb(184, 69, 69);
+    list-style: none;
+    
+  }
+
+ul {
+    display: flex;
+    margin: 0;
+    list-style: none;
+    text-align: end;
+    color: white;
+}
+
+li {
+    padding: 20px;
+    font-size: large;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+#welcome-section {
+    border: 4px solid brown;
+    height: 100vh;
+    overflow: auto;
+    background-color: rgb(73, 73, 87);
+    margin: auto;
+    
+}
+
+.welcome {
+    border: 4px solid blue;
+    align-items: center;
+    justify-content: center;
+    text-align: center; 
+    margin-top: 300px; 
+}
+
+
+
+h2 {
+    align-items: center;
+    justify-content: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 48px;
+    color: white;
+}
+
+h3 {
+    font-size: 36px;
+    color: red;
+    font-style: italic;
+}
+
+.project-section {
+    display: flex;
+}
+
+.grid-wrapper {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
+    grid-gap: 20px;
+    background-color: rgb(184, 69, 69);
+}
+
+.project-tile {
+    background-color: #444;
+    padding: 20px;
+    margin: 20px;
+}
+
+.project-tile > img {
+    width: 100%;
+}
+
+
+.project-tile > img {
+    width: 100%;
+    height: auto;
+}
+
+.project-tile > a {
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+}
+
+footer {
+    text-align: center;
+    background-color: #444;
+    height: 200px;
+    justify-content: center;
+    font-size: 36px;
+
+}
+
+footer > a {
+    
+    margin-top: 80px;
+    color: white;
+}
```

Comment:
You did a good job with the CSS. I suggest reviewing the linters comments and solving each one, it will help you to improve the quality of your code. For example, now I'm selecting the final part of your CSS code, and linters ask to add an empty end of line, that's mean when you press enter in the last line to add an empty line you will solve this comment. Proceed in a similar way with the other comments.

Reply:
✅ Good job pointing that, it seems like a professional review ^_^